pyomo/contrib/parmest/parmest.py

```diff
@@ -719,14 +1337,14 @@ def _Q_at_theta(self, thetavals, initialize_parmest_model=False):
                             theta_init_vals.append(var_validate)
                         except:
                             logger.warning(
-                                'Unable to fix model parameter value for %s (not a Pyomo model Var)',
+                                "Unable to fix model parameter value for %s (not a Pyomo model Var)",
                                 (theta),
                             )
 
             if active_constraints:
                 if self.diagnostic_mode:
-                    print('      Experiment = ', snum)
-                    print('     First solve with special diagnostics wrapper')
+                    print("      Experiment = ", snum)
+                    print("     First solve with special diagnostics wrapper")
```

Comment:
It looks like you may have not correctly applied `black.` Reminder that we use `black -S -C .` when running `black` - this stops it from changing `'` to `"`